tests/test_consultation_request_ownership.py

```
@@ -0,0 +1,146 @@
+        assert "consultation_request_id" not in state.data
+
+    asyncio.run(scenario())
+
+
+def test_slow_consultation_is_suppressed_after_document_switch(monkeypatch: pytest.MonkeyPatch) -> None:
+    async def scenario() -> None:
+        started = asyncio.Event()
+        release = asyncio.Event()
+
+        class Service:
+            async def consult(self, question: str, *, case_context: str, language: str):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `consult`

(ANN202)